tests/e2e/utils/utils.ts

```diff
@@ -0,0 +1,13 @@
+export function createPackageJson(filePath: string) {
+  let pkgJson = {
+    name: 'test-package',
+    version: '0.0.1',
+    description: 'dummy package',
+    main: 'index.js',
+    scripts: {},
+    author: 'admin',
+    license: '',
+    dependencies: {}
+  };
+  return fs.writeFile(filePath, JSON.stringify(pkgJson));
```

Comment:
break line before return please, and `JSON.stringify(pkgJson, null, 2)` has nicer output.